qutip/visualization.py

```diff
@@ -246,17 +268,18 @@ def hinton(rho, x_basis=None, y_basis=None, color_style="scaled",
         list of y ticklabels to represent y basis of the input.
 
     color_style : string, default="scaled"
+
         Determines how colors are assigned to each square:
 
         -  If set to ``"scaled"`` (default), each color is chosen by
-           passing the absolute value of the corresponding matrix
-           element into `cmap` with the sign of the real part.
+        passing the absolute value of the corresponding matrix
+        element into `cmap` with the sign of the real part.
         -  If set to ``"threshold"``, each square is plotted as
-           the maximum of `cmap` for the positive real part and as
-           the minimum for the negative part of the matrix element;
-           note that this generalizes `"threshold"` to complex numbers.
+        the maximum of `cmap` for the positive real part and as
+        the minimum for the negative part of the matrix element;
+        note that this generalizes `"threshold"` to complex numbers.
         -  If set to ``"phase"``, each color is chosen according to
-           the angle of the corresponding matrix element.
+        the angle of the corresponding matrix element.
```

Comment:
I think it's these bullet list that cause the issues with the documentation.
The indentation is needed.